myhpi/tenca_django/wagtail_hooks.py

```diff
@@ -0,0 +1,24 @@
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel
+from wagtail.snippets.models import register_snippet
+from wagtail.snippets.views.snippets import SnippetViewSet
+
+from myhpi.tenca_django.models import HashEntry
+
+
+class HashEntryFilterSet(SnippetViewSet):
+    model = HashEntry
+    add_to_admin_menu = True
+    menu_label = _("Mailing Lists")
```

Comment:
This does not seem to work for me
![image](https://github.com/fsr-de/myHPI/assets/6863832/fadeec33-c387-4787-9992-480d99a307a6)

Reply:
@frcroth are you fine with the way it is?

Reply:
sure